src/main/java/mate/academy/repository/book/BookSpecificationBuilder.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.repository.book;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.dto.BookSearchParametersDto;
+import mate.academy.model.Book;
+import mate.academy.repository.SpecificationBuilder;
+import mate.academy.repository.SpecificationProviderManager;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class BookSpecificationBuilder implements SpecificationBuilder<Book> {
+    private static final String TITLE = "title";
+    private static final String AUTHOR = "author";
+    private static final String ISBN = "isbn";
```

Comment:
Let’s make constants public and reuse them. (better have one place for future changes)